plugins/node/opentelemetry-instrumentation-aws-lambda/src/instrumentation.ts

```diff
@@ -228,7 +227,32 @@ export class AwsLambdaInstrumentation extends InstrumentationBase {
 
   override setTracerProvider(tracerProvider: TracerProvider) {
     super.setTracerProvider(tracerProvider);
-    this._tracerProvider = tracerProvider;
+    this._forceFlush = this._getForceFlush(tracerProvider);
+  }
+
+  private _getForceFlush(tracerProvider: TracerProvider) {
+    if (!tracerProvider) {
+      diag.error(
```

Comment:
not sure if `error` is the correct log level here because a user may decide to disable the sdk for whatever reason.
Or in other words: it's not an error of this instrumentation and nothing this instrumentation can change.

Reply:
You are right. Now that I refactored and this is happening during the setting of the tracer provider and not during the end span, this log does not make sense. Changing it to debug.

Reply:
Actually, I'm going to get rid of it as it doesn't make sense and hard to follow.